src/application/Types/SearchPersonTypes.ts

```diff
@@ -0,0 +1,7 @@
+export interface SearchPersonType {
+    personID:number;
```

Comment:
nitpick: `personId` would be better convention.

Reply:
with this refactor, i thought xxxID looked better so i turned every ID elements to this, but this is ofc reversible